### DISCUSSION TITLE
Quiz: 2.4 - Part 3: Multiplying Two Matrices

### DISCUSSION THREAD

**Discussion by rishok** [Lightning-AI/dl-fundamentals#51] [Lecture videos and quizzes] [answered]
I'm not sure if the numbers from the hint are a typo. Should not it be: 1x6 + 2x8 = 22 ? 

![Screen Shot 2023-05-12 at 10 23 20](https://github.com/Lightning-AI/dl-fundamentals/assets/4780216/c7e7b55d-ec30-4239-89db-8b12ce6b489f)

**Comment by jihobak** [comment, marked as answer]
Yeah, right

**Comment by rasbt** [comment]
Omg the hint was indeed totally messed up! Thanks for the note, just fixed it!
